bahmnicore-omod/src/test/java/org/bahmni/module/bahmnicore/web/v1_0/search/BacteriologySpecimenSearchHandlerIT.java

```diff
@@ -2,14 +2,16 @@
 
 import org.junit.Assert;
 import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
 import org.openmrs.module.webservices.rest.SimpleObject;
 import org.openmrs.module.webservices.rest.web.RestConstants;
 import org.springframework.mock.web.MockHttpServletRequest;
 import org.springframework.web.bind.annotation.RequestMethod;
 
 import java.util.List;
-
+//Bacteriology OMOD needs to be updated with OpenMRS 2.2.* version
```

Comment:
This requires change in the bacteriology OMOD as mentioned in the comment. Once this is done we can remove the @Ignore statement.

Reply:
Removed @ignore as part of 2.3.0 OMRS upgrade !